Lib/importlib/abc.py

```diff
@@ -15,20 +15,36 @@
 import abc
 import warnings
 
-# for compatibility with Python 3.10
-from .resources.abc import ResourceReader, Traversable, TraversableResources
+from .resources import abc as _resources_abc
 
 
 __all__ = [
     'Loader', 'Finder', 'MetaPathFinder', 'PathEntryFinder',
     'ResourceLoader', 'InspectLoader', 'ExecutionLoader',
     'FileLoader', 'SourceLoader',
-
-    # for compatibility with Python 3.10
-    'ResourceReader', 'Traversable', 'TraversableResources',
```

Comment:
This is technically a backward incompatibility though, right?  Shouldn't these names continue to be available in `__all__` until removal in 3.13?

Reply:
I'm following the pattern as was used in `collections` to deprecate its abcs. In that case, the names were [removed from `__all__` at the time of deprecation]. My primary reason for removing them was because the linter complains when there are names in `__all__` that don't exist in the module. The only users this will affect are those using `import *`, which itself is discouraged, so I propose to leave them out, follow the prior pattern, and if it turns out to impact anyone, we can restore the names.